testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/policy/PasswordPolicyTest.java

```diff
@@ -175,6 +175,19 @@ public void testNotUsername() {
         });
     }
 
+    @Test
+    public void testNotPersonalInformation() {
+        testingClient.server("passwordPolicy").run(session -> {
+            RealmModel realmModel = session.getContext().getRealm();
+            PasswordPolicyManagerProvider policyManager = session.getProvider(PasswordPolicyManagerProvider.class);
+
+            realmModel.setPasswordPolicy(PasswordPolicy.parse(session, "notPersonalInformation"));
+            Assert.assertEquals("invalidPasswordNotPersonalInformationMessage", policyManager.validate("jdoe", "jdoe").getMessage());
```

Comment:
I'd like to see some more tests to include checking for everything that is not permitted (firstname, lastname, email, email exploded) and also checking for upper/lower case.